core/data.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	commonpb "github.com/Layr-Labs/eigenda/api/grpc/common"
 	"github.com/Layr-Labs/eigenda/common"
+	paymentvault "github.com/Layr-Labs/eigenda/contracts/bindings/PaymentVault"
```

Comment:
can we not have contract dependency on core data structs?

Reply:
can we do something here?

Reply:
what do you think about moving `type ActiveReservation = paymentvault.IPaymentVaultReservation` to core/eth?

Reply:
How does that remove the contract dependency?

Reply:
core/eth already has contract dependency, so we are won't introducing it in data.go, we just need to avoid circular dependencies in this case